backend/crypt/cipher.go

```diff
@@ -191,7 +191,7 @@ func newCipher(mode NameEncryptionMode, password, salt string, dirNameEncrypt bo
 		dirNameEncrypt: dirNameEncrypt,
 	}
 	c.buffers.New = func() interface{} {
-		return make([]byte, blockSize)
```

Comment:
ingenious using an array rather than a slice. This definitely uses less allocations :+1: